test/integration/auth/node_test.go

```diff
@@ -248,6 +275,9 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectAllowed(t, createNode2(node2Client))
 	expectAllowed(t, updateNode2Status(node2Client))
 	expectAllowed(t, deleteNode2(node2Client))
+	// node evicting pod bound to itself is allowed
+	expectAllowed(t, createNode2MirrorPod(node2Client))
+	expectAllowed(t, createNode2MirrorPodEviction(node2Client))
```

Comment:
Got it. In this case, the removed mirror pod will be recreated. So, if the pod is recreated, aren't the subsequent Forbidden errors to be expected? (We cannot expect NotFound errors because those pods are being recreated afterall, right?)

Sorry for the incessant questions. I just want to make sure I get this right!